src/cli/cli.ts

```diff
@@ -23,12 +25,21 @@ export const runCli = async () => {
     return process.stdout.write(getHelpText.run());
   }
 
+  const outputCjs = args.outputType === "CJS" || 
```

Comment:
We might want to seperate the outputtype from the outputfile ending. Depends on the outcome of the trview comment I made on the flag.

Reply:
i think basing it on the extension type is easier to use, generally speaking. also, if you set output-type = TS & the output file to something.js, using TS syntax would be invalid so the output-type should get overriden.

otoh, if you set output-type = JS & the output as something.ts, that doesnt have any significant issues, although eslint might complain.